README.md

```diff
@@ -123,26 +122,24 @@ fn main() {
 use elkodon::prelude::*;
 use elkodon_bb_posix::signal::SignalHandler;
 
-fn main() {
-    let service_name = ServiceName::new(b"My/Funk/ServiceName").unwrap();
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let service_name = ServiceName::new(b"My/Funk/ServiceName")?;
 
     let service = zero_copy::Service::new(&service_name)
         .publish_subscribe()
-        .open_or_create::<TransmissionData>()
-        .expect("failed to create/open service");
+        .open_or_create::<usize>()?;
```

Comment:
We need geoffrey for Rust. This did not compile :grimacing:

Reply:
I created an issue for this